practice-01/index.html

```diff
@@ -290,16 +326,102 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
+                        </li>
+                        <li>
+                            <img src="./img/Linkedin.png" alt="Here is a photo of the media above" class="media-icon">
+                        </li>
+                    </ul>
+
             </div>
-            <div class="footer__rules"></div>
+            <div class="content-right">
+                <div class="footer__list">
+                    <ul class="list-menu">
+                        <li class="item ">Company</li>
+                        <li class="item">About</li>
+                        <li class="item">Pricing</li>
+                        <li class="item">Jobs</li>
+                        <li class="item">Blog</li>
+                        <li class="item">Careers</li>
+                    </ul>
+                </div>
+                <div class="footer__list">
+                    <ul class="list-menu">
+                        <li class="item ">Product</li>
+                        <li class="item">Sales software</li>
+                        <li class="item">Features</li>
+                        <li class="item">Privacy and security</li>
+                        <li class="item">Marketplace</li>
+                        <li class="item">Status</li>
+                        <li class="item">API</li>
+                    </ul>
+                </div>
+                <div class="footer__list">
+                    <ul class=" list-menu">
+                        <li class="item ">Discover</li>
+                        <li class="item">Partner Program</li>
+                        <li class="item">Get our newsletter</li>
+                        <li class="item">Sales Pipeline Course</li>
+                        <li class="item">What is CRM?</li>
+                        <li class="item">CRM Comparison</li>
+                        <li class="item">Resources</li>
+                    </ul>
+                </div>
+                <div class="footer__list">
+                    <ul class="list-menu">
+                        <li class="item ">Help Center</li>
+                        <li class="item">Community</li>
+                        <li class="item">Knowledge Base </li>
+                        <li class="item">Academy</li>
+                        <li class="item">Support</li>
+                    </ul>
+                </div>
+            </div>
+
+        </div>
+
+        <hr class="footer__line">
+
+        <div class="footer__rule container">
+            <a class="link-mail">&#169;Copyright 2022 Yoora, Inc.</a>
+
+            <div class="footer__rule-block">
+                <ul class="privacy-list">
+                    <li class="item">Terms of Service</li>
+                    <li class="item">Privacy Policy</li>
+                    <li class="item">Cookies</li>
+                </ul>
+                <div class="locale-switche">
+                    <button class="btn btn--switcher">
```

Comment:
btn--switcher => bad name. It should relative to css in this case. let try `btn--flex`